src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -167,7 +170,7 @@ public SAMSequenceDictionary makeSequenceDictionary(final File referenceFile) {
     private Iterable<SAMSequenceRecord> getSamSequenceRecordsIterable() {
         return () -> {
             final SequenceDictionaryUtils.SamSequenceRecordsIterator iterator =
-                    new SequenceDictionaryUtils.SamSequenceRecordsIterator(REFERENCE_SEQUENCE,
+                    new SequenceDictionaryUtils.SamSequenceRecordsIterator(referenceSequence.getHtsPath().toPath(),
```

Comment:
Why not use 'referenceSequence.getReferencePath()' instead of duplicating the 'toPath()' call here.

Reply:
Will do

Reply:
Done